test/classes/initializers/inheritance/inheritedInitWrongName.good

```diff
@@ -0,0 +1,4 @@
+inheritedInitWrongName.chpl:20: error: unresolved call '_new(type D, x=3, "hi", 4.6)'
+inheritedInitWrongName.chpl:1: note: candidates are: _new(type chpl_t, x, y, z)
+inheritedInitWrongName.chpl:14: note:                 _new(type chpl_t, a, b, c)
+deleted module lines
```

Comment:
I don't think there should be an ambiguity error here. The source code clearly says it's creating a `D` : `new D ( ... )`. So it should be calling a `D` initializer *no matter what the arguments are*.

Reply:
The `new D(...)` call uses named argument passing for `x`, but the `D` initializer only has formals `a, b, c`.  I think `unresolved call` makes sense in that case.

Reply:
Quite right, quite right. I stand corrected.

Reply:
I think that means that the test file should fail no matter what, yes?

Reply:
Right, it shouldn't compile.

Reply:
As our compiler gets better at printing out the actual candidates, I think this .good file output will change (improve), but this was my best guess as to what it should be in the immediate-ish term (if this issue were fixed in the next week or two, say).